src/views/Map/CustomerMap.js

```diff
@@ -0,0 +1,221 @@
+
+import React from 'react'
+import { AppSwitch } from '@coreui/react'
+import { Input, Row, FormGroup } from 'reactstrap';
+
+var map, infoWindow, marker, markers, geoCodeValue;
+var google = window.google;
+
+class CustomerMap extends React.PureComponent {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      searchBox: '',
+      addressBox: '',
+      checked: true
+    }
+    this.googleMapRef = React.createRef()
+    this.initAutocomplete = this.initAutocomplete.bind(this)
+    this.handleChange = this.handleChange.bind(this)
+    this.handleMapChange = this.handleMapChange.bind(this)
+
+  }
+
+  handleMapChange(value) {
+    this.setState({
+      addressBox: value
+    })
+  }
+
+  handleChange(e) {
+    switch (e.target.id) {
+      case 'pac-input':
+        this.setState({
+          searchBox: e.target.value
+        })
+        break;
+      case 'customer_address':
+        this.setState({
+          addressBox: e.target.value
+        })
+        break;
+      case 'address_switch':
+        this.setState({
+          checked: e.target.checked
+        })
+        break;
+      default:
+        break;
+    }
+  }
+
+  componentDidMount() {
+    let googleScript = document.createElement('script');
+    // API key invalid, add yours or contact @johnerry on git
+    googleScript.src = "https://maps.googleapis.com/maps/api/js?key=AIzatSyBKHtdFymwut91lTBTQBDgltrFaXPD6TxY2g&libraries=places"
+    googleScript.async = true
+    window.document.body.appendChild(googleScript)
+    googleScript.addEventListener('load', this.initAutocomplete)
+  }
+
+  initAutocomplete() {
+    let mapToState = this.handleMapChange
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['mapToState' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)